dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/utils/ArchiveNameUtil.java

```diff
@@ -36,6 +40,17 @@ public static String getFileNameWithTimestamp(String name, Clock clock) {
     return formatFileName(name, timeSuffix);
   }
 
+  public static String getEntryFileNameWithTimestamp(String prefix, ZonedInterval interval) {
+    return getEntryFileNameWithTimestamp(prefix, interval, ZIP_ENTRY_SUFFIX);
+  }
+
+  public static String getEntryFileNameWithTimestamp(
```

Comment:
Let's add a unit-test for the method with some explanation. To make sure that in 2-3 years from now someone will not decide to change the format back.

Reply:
done, I added simple unit test for this.